lib/api/controllers/routerController.js

```diff
@@ -54,7 +54,7 @@ module.exports = function RouterController (kuzzle) {
       executeFromRest.call(kuzzle, params, request, response);
     });
 
-    api.get('/_getStats', function (request, response) {
+    api.post('/_getStats', function (request, response) {
```

Comment:
This would mean we want a stat with that exact timestamp but the purpose of this is to get the stats SINCE that timestamp so i thought it's better to use POST